documentation/assemblies/configuring/assembly-scaling-kafka-clusters.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// deploying.adoc
+
+[id='assembly-scaling-kafka-clusters-{context}']
+= Scaling clusters by adding or removing brokers
+
+[role="_abstract"]
+Scaling Kafka clusters by adding brokers can improve performance and reliability. 
+Increasing the number of brokers provides more resources, enabling the cluster to handle larger workloads and process more messages. 
+It also enhances fault tolerance by providing additional replicas. 
+Conversely, removing underutilized brokers can reduce resource consumption and increase efficiency. 
+Scaling must be done carefully to avoid disruption or data loss. 
+Redistributing partitions across brokers reduces the load on individual brokers, increasing the overall throughput of the cluster.
+
+Adjusting the `Kafka.spec.kafka.replicas` configuration changes the number of brokers in the cluster. 
+The actual replication factor for topics is determined by the `default.replication.factor` and `min.insync.replicas` settings, and the number of available brokers. 
+
+A replication factor of 3 means each partition is replicated across three brokers, ensuring fault tolerance in case of a broker failure:
+
+.Example replica configuration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
```

Comment:
Perhaps not a change for this PR, but in future we should switch this example to show how to do it with KafkaNodePools since that will be the default way when ZooKeeper is removed.

Reply:
Thanks @katheris 
No time like the present.  I've updated that introduction to show node pool replicas config and kafka config examples.
The surrounding text should be future-proof now too.
If you could please check the change, that would be great:  [docs(review): updates to doc from review by KS](https://github.com/strimzi/strimzi-kafka-operator/pull/10698/commits/6365aa5e7119d0bb546d2792522b939238a7e4d1)